docs/modules/operation/pages/geographical-map/searching.adoc

```diff
@@ -0,0 +1,73 @@
+## Searching
+
+The search bar has the ability to do a simple text search, as well as searching particular fields associated with the node.
+
+### Search Fields
+
+The following fields can be searched:
+
+[options="header"]
+[cols="2*"]
+|====
+| Field
+| Description
+
+| `nodeId`
```

Comment:
```suggestion
| nodeId
```
Please remove the `` characters for all of the content in the field columns. We don't need to distinguish it typographically.

Reply:
fixed with https://github.com/OpenNMS/opennms/pull/4351/commits/9059d24fb12d76b613bda9362bf0fa22dcfaf45b